### PR TITLE
Load synonyms

### PR DESCRIPTION
**Pull request by cmpich**

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/model/entities/Synonym.java [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
 @Entity
-@Data @EqualsAndHashCode(callSuper = false)
+@Data
+@EqualsAndHashCode(callSuper = false)
 public class Synonym extends BaseGeneratedEntity {
 
+    @JsonView({View.FieldsOnly.class})
     private String name;
 
+    public Synonym(String name) {
```
[the remark this answers is not in the document]
Comment (cmpich) [reply src/main/java/org/alliancegenome/curation_api/model/entities/Synonym.java:23]: Ah, ok. Forgot.

[the remark this answers is not in the document]
Comment (cmpich) [reply src/main/java/org/alliancegenome/curation_api/model/entities/Synonym.java:23]: Will remove it.

[the remark this answers is not in the document]
Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/model/entities/Synonym.java:23]: @cmpich I am wrong... the better way is using: @AllArgsConstructor and @NoArgsConstructor

File: src/main/java/org/alliancegenome/curation_api/base/BaseCrudRESTInterface.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
     @GET //@Secured
     @Path("/{id}")
-    @JsonView(View.FieldsOnly.class)
+    @JsonView(View.FieldsAndLists.class)
```
[the remark this answers is not in the document]
Comment (cmpich) [reply src/main/java/org/alliancegenome/curation_api/base/BaseCrudRESTInterface.java:23]: Then I have to change the JsonView on the 'synonyms' list in GenomicEntity class to also use FieldsOnly view.

File: src/main/java/org/alliancegenome/curation_api/model/entities/Synonym.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (cmpich) [reply src/main/java/org/alliancegenome/curation_api/model/entities/Synonym.java:23]: Actually, @Data does not create the one ion question but an allArgs one with the genomicEntityList included.

File: src/main/java/org/alliancegenome/curation_api/base/BaseCrudRESTInterface.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/base/BaseCrudRESTInterface.java:23]: Well... The other way to do this is a specific get method on the controller "GetFullGene" that uses a different View... that way the "default" view doesn't cause issues.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**